shared/actions/__tests__/wallets.test.js

```diff
@@ -149,3 +149,41 @@ it('build and send payment', () => {
       expect(sendRPC).toHaveBeenCalled()
     })
 })
+
+const buildRequestRes: RPCStellarTypes.BuildRequestResLocal = {
+  amountErrMsg: '',
+  banners: null,
+  displayAmountFiat: '$5.00 USD',
+  displayAmountXLM: '21.4168160 XLM',
+  readyToRequest: false,
+  secretNoteErrMsg: '',
+  sendingIntentionXLM: false,
+  toErrMsg: '',
+  worthDescription: '21.4168160 XLM',
+  worthInfo: '$1.00 = 4.2833632 XLM\nSource: coinmarketcap.com',
+}
+
+it('build and send request', () => {
+  const {dispatch, getState} = startReduxSaga()
+  const buildRPC = jest.spyOn(RPCStellarTypes, 'localBuildRequestLocalRpcPromise')
+  buildRPC.mockImplementation(() => new Promise(resolve => resolve(buildRequestRes)))
```

Comment:
you can use `Promise.resolve(buildRequestRes)` instead

Reply:
Cool! Also fixed a bunch of other places in the tests.